frontend/src/app/general/api-documentation/api-documentation.component.html

```diff
@@ -0,0 +1,14 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="pb-3 text-xs">
+  Check our
+  <a class="text-url" [href]="getAPIDocsLink()" target="_blank"
+    >API documentation<mat-icon inline class="ml-1 align-text-bottom"
+      >open_in_new</mat-icon
+    ></a
+  >
+  for available options.
+</div>
```

Comment:
> I'd generally avoid text-xs (12px) and use text-sm (14px) instead, and still sparingly.

Changed it to `text-sm`.

Reply:
> You're using pb-3 both here and in the app-api-documentation tag, but I'd remove it from both. I wouldn't keep it in this file because you're concerned with spacing relative to other elements, so it's easier to get the full picture in tool-details-component.html in this case. I'm still sorting this out, but from what I'm seeing inline classes on custom components don't actually do anything (I think Angular skips them while constructing the dom and they need be included in the TypeScript?). So what makes the most sense to me if we're keeping everything inline is actually wrapping the app-api-documentation tag in something else to provide the spacing you need.

Yes, makes sense. Removed the `pb-3` from the component and added div-wrappers around the calls to the component. Also removed the useless inline styles on the <app-...> tag.

Reply:
> And a small distinction, I'd use margin rather than padding, because the concern is spacing relative to other componets and not spacing within the border of the component itself.

Changed `pb-3` to `mb-3`